python/src/HSICEstimatorImplementation_doc.i.in

```diff
@@ -0,0 +1,284 @@
+
+%define OT_HSICEstimator_getEstimator_doc
+"Get the estimator used for computations.
+
+Returns
+-------
+estimator : :class:`~openturns.HSICStat`
+            The estimator used for internal computations."
+%enddef
+%feature("docstring") OT::HSICEstimatorImplementation::getEstimator
+OT_HSICEstimator_getEstimator_doc
+
+// ---------------------------------------------------------------------
+
+%define OT_HSICEstimator_getHSICIndices_doc
+"Get the HSIC indices.
+
+Returns
+-------
+hsicIndices : :class:`~openturns.Point`
+              The HSIC indices of all components."
+%enddef
+%feature("docstring") OT::HSICEstimatorImplementation::getHSICIndices
+OT_HSICEstimator_getHSICIndices_doc
+
+// ---------------------------------------------------------------------
+
+%define OT_HSICEstimator_getR2HSICIndices_doc
+"Get the R2-HSIC indices.
+
+Returns
+-------
+r2hsicIndices : :class:`~openturns.Point`
+                The R2-HSIC indices of all components."
+%enddef
+%feature("docstring") OT::HSICEstimatorImplementation::getR2HSICIndices
+OT_HSICEstimator_getR2HSICIndices_doc
+
+// ---------------------------------------------------------------------
+
+%define OT_HSICEstimator_getPValuesPermutation_doc
+"Get the p-value estimated through permutations.
+
+Returns
+-------
+pval : :class:`~openturns.Point`
+       The p-values of all components estimated with permutations of the data."
+%enddef
+%feature("docstring") OT::HSICEstimatorImplementation::getPValuesPermutation
+OT_HSICEstimator_getPValuesPermutation_doc
+
+// ---------------------------------------------------------------------
+
+%define OT_HSICEstimator_drawHSICIndices_doc
+"Draw the HSIC indices.
+
+Returns
+-------
+graph : :class:`~openturns.Graph`
+        The graph of all HSIC indices according to components."
+%enddef
+%feature("docstring") OT::HSICEstimatorImplementation::drawHSICIndices
+OT_HSICEstimator_drawHSICIndices_doc
+
+// ---------------------------------------------------------------------
+
+%define OT_HSICEstimator_drawR2HSICIndices_doc
+"Draw the R2-HSIC indices.
+
+Returns
+-------
+graph : :class:`~openturns.Graph`
+        The graph of all R2-HSIC indices according to components."
+%enddef
+%feature("docstring") OT::HSICEstimatorImplementation::drawR2HSICIndices
+OT_HSICEstimator_drawR2HSICIndices_doc
+
+// ---------------------------------------------------------------------
+
+%define OT_HSICEstimator_drawPValuesPermutation_doc
+"Draw the p-values obtained by permutation.
+
+Returns
+-------
+graph : :class:`~openturns.Graph`
+        The graph of all p-values by permutation according to components."
+%enddef
+%feature("docstring") OT::HSICEstimatorImplementation::drawPValuesPermutation
+OT_HSICEstimator_drawPValuesPermutation_doc
+
+// ---------------------------------------------------------------------
+
+%define OT_HSICEstimator_drawValues_doc
+"Draw values contained in a point.
+
+Parameters
+----------
+values : :class:`~openturns.Point`
```

Comment:
on input Points can be a "sequence of float"

Reply:
OK